config/.azure/commandIndex.json

```diff
@@ -0,0 +1 @@
"], "maps": ["azure.cli.command_modules.maps"], "term": ["azure.cli.command_modules.marketplaceordering"], "monitor": ["azure.cli.command_modules.monitor"], "mysql": ["azure.cli.command_modules.mysql", "azure.cli.command_modules.rdbms"], "netappfiles": ["azure.cli.command_modules.netappfiles"], "network": ["azure.cli.command_modules.network", "azure.cli.command_modules.privatedns"], "policy": ["azure.cli.command_modules.policyinsights", "azure.cli.command_modules.resource"], "login": ["azure.cli.command_modules.profile"], "logout": ["azure.cli.command_modules.profile"], "self-test": ["azure.cli.command_modules.profile"], "account": ["azure.cli.command_modules.profile", "azure.cli.command_modules.resource"], "mariadb": ["azure.cli.command_modules.rdbms"], "postgres": ["azure.cli.command_modules.rdbms"], "redis": ["azure.cli.command_modules.redis"], "relay": ["azure.cli.command_modules.relay"], "data-boundary": ["azure.cli.command_modules.resource"], "group": ["azure.cli.command_modules.resource"], "resource": ["azure.cli.command_modules.resource"], "provider": ["azure.cli.command_modules.resource"], "feature": ["azure.cli.command_modules.resource"], "tag": ["azure.cli.command_modules.resource"], "deployment": ["azure.cli.command_modules.resource"], "deployment-scripts": ["azure.cli.command_modules.resource"], "ts": ["azure.cli.command_modules.resource"], "stack": ["azure.cli.command_modules.resource"], "lock": ["azure.cli.command_modules.resource"], "managedapp": ["azure.cli.command_modules.resource"], "bicep": ["azure.cli.command_modules.resource"], "resourcemanagement": ["azure.cli.command_modules.resource"], "private-link": ["azure.cli.command_modules.resource"], "role": ["azure.cli.command_modules.role"], "ad": ["azure.cli.command_modules.role"], "search": ["azure.cli.command_modules.search"], "security": ["azure.cli.command_modules.security"], "servicebus": ["azure.cli.command_modules.servicebus"], "connection": ["azure.cli.command_modules.serviceconnector"], "sf": ["azure.cli.command_modules.servicefabric"], "signalr": ["azure.cli.command_modules.signalr"], "sql": ["azure.cli.command_modules.sql", "azure.cli.command_modules.sqlvm"], "storage": ["azure.cli.command_modules.storage"], "synapse": ["azure.cli.command_modules.synapse"], "rest": ["azure.cli.command_modules.util"], "version": ["azure.cli.command_modules.util"], "upgrade": ["azure.cli.command_modules.util"], "demo": ["azure.cli.command_modules.util"], "snapshot": ["azure.cli.command_modules.vm"], "disk-access": ["azure.cli.command_modules.vm"], "sig": ["azure.cli.command_modules.vm"], "vmss": ["azure.cli.command_modules.vm"], "restore-point": ["azure.cli.command_modules.vm"], "image": ["azure.cli.command_modules.vm"], "capacity": ["azure.cli.command_modules.vm"], "vm": ["azure.cli.command_modules.vm"], "disk": ["azure.cli.command_modules.vm"], "ppg": ["azure.cli.command_modules.vm"], "disk-encryption-set": ["azure.cli.command_modules.vm"], "sshkey": ["azure.cli.command_modules.vm"]}}
```

Comment:
Not sure why this was generated? It doesn't appear in the latest generation. https://github.com/databricks/databricks-sdk-go/pull/1221